packages/kit/test/apps/basics/src/routes/encoded/%5Funderscore-prefix/+page.svelte

```diff
@@ -0,0 +1 @@
+<h1>_underscore-prefix</h1>
```

Comment:
this file is broken. if you want to use an underscore prefix then just use an underscore. there's no need to encode it